src/VideoNativeComponent.ts

```diff
@@ -148,12 +148,15 @@ export type OnProgressData = Readonly<{
   seekableDuration: number;
 }>;
 
-export type OnBandwidthUpdateData = Readonly<{
-  bitrate: number;
-  width: number;
-  height: number;
-  trackId: number;
-}>;
+export type OnBandwidthUpdateData = Readonly<
+  | {
+      bitrate: number;
+      width: number;
```

Comment:
can it be clearer to define it as width?: number ?

Reply:
Here must be - forgot that code gen don't support types like this. Will fix this

Reply:
Done ✅